install_scripts/install_lago.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -xe
 
+#Let's reviews the install script
 
 REPONAME="ovirt-system-tests"
 REPOURL="https://gerrit.ovirt.org"
```

Comment:
Because the script must run under root, I don't see how 'newgrp' can help here.
Maybe I'll run "su - $username" at the end of the script ? (I'm getting the username as a positional parameter)